p8e-migration/sql/V67__Add_Auth_Public_Key_to_Affiliate.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE affiliate ADD COLUMN auth_public_key TEXT;
```

Comment:
@rchaing-figure should we go ahead and have this migration just copy over the signing_public_key value for existing affiliates and make this column not nullable?

Reply:
Wasn't ready to do this yet until I get the Encryption Key all setup as well, then see how this all looks and see which key would make more sense.